src/Components/TabsPage/TabsPage.tsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import { useParams } from 'react-router-dom';
+import { Tab } from '../../types/Tab';
+import { Tabs } from '../Tabs/Tabs';
+
+type Props = {
+  tabs: Tab[];
+};
+
+export const TabsPage: React.FC<Props> = ({
+  tabs,
+}) => {
+  const { tabId } = useParams();
+
```

Comment:
You don't use tabId in this component. You need it only in your Tabs, so move useParams to the Tabs component